docs/versioned-plugins/include/6.x/stack_version.asciidoc

```diff
@@ -0,0 +1,18 @@
+ifeval::["{versioned_docs}"=="true"]
+:branch: %BRANCH%
+:ecs_version: %ECS_VERSION%
+endif::[]
```

Comment:
Add to plugin header file?

Reply:
Another idea: Add a version include file for each stack/ecs version.  Use cond'l logic to inject attributes only for VPR.